src/commands/assets/registerAsset.ts

```diff
@@ -27,42 +27,37 @@ export const registerAsset = async (
   config: ConfigEntry,
   logger: Logger
 ): Promise<ExecutionOutput> => {
-  const { verbose, metadata, assetType, password } = argv
+  const { verbose, metadata, assetType, _password } = argv
   const token = await loadToken(nvm, config, verbose)
 
   // TODO: Enable DTP when `sdk-dtp` is ready
-  const keyTransfer = await makeKeyTransfer()
-
+  const password = '4e657665726d696e65640a436f707972696768742032303230204b65796b6f20476d62482e0a0a546869732070726f6475637420696e636c75646573'
   if (verbose) {
     printTokenBanner(token)
   }
 
   logger.info(chalk.dim(`Registering asset (${assetType}) ...`))
 
-  logger.info(JSON.stringify(argv))
-
   logger.debug(chalk.dim(`Using creator: '${account.getId()}'\n`))
 
   let ddoMetadata: MetaData
 
   const ddoPrice = BigNumber.from(argv.price).gt(0)
     ? BigNumber.from(argv.price)
     : BigNumber.from(0)
```

Comment:
I don't think this piece of code is doing what's expected. `BigNumber.from` will fail if `argv.price` is not set

Reply:
`argv.price` has a default value of `0` if is not provided by user